src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/wwwroot/css/site.css

```diff
@@ -8,6 +8,15 @@ a, .btn-link {
     color: #0366d6;
 }
 
+::-moz-selection {
```

Comment:
According to https://www.caniuse.com/?search=%3A%3Aselection, the last version of FireFox that needed this was released in 2018. Versions since support the `::selection` selector. We should be able to remove this.

Reply:
Cool will update